File: app/src/main/java/com/tammamkhalaf/myuaeguide/User/AllCategories.java
package com.tammamkhalaf.myuaeguide.User;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.view.WindowManager;
import android.widget.ImageView;

import com.tammamkhalaf.myuaeguide.R;

public class AllCategories extends AppCompatActivity {

    ImageView back_btn;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_all_categories);

        getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN,WindowManager.LayoutParams.FLAG_FULLSCREEN);

        back_btn = findViewById(R.id.back_pressed);

        back_btn.setOnClickListener(view -> {
            AllCategories.super.onBackPressed();
        });

    }
}

File: app/src/main/java/com/tammamkhalaf/myuaeguide/User/AllCategories.kt
package com.tammamkhalaf.myuaeguide.User

import android.os.Bundle
import android.view.View
import android.view.WindowManager
import android.widget.*
import androidx.appcompat.app.AppCompatActivity
import com.tammamkhalaf.myuaeguide.R
import com.tammamkhalaf.myuaeguide.User.AllCategories

class AllCategories : AppCompatActivity() {
    lateinit var back_btn: ImageView
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        window.setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN, WindowManager.LayoutParams.FLAG_FULLSCREEN)
        setContentView(R.layout.activity_all_categories)
        back_btn = findViewById(R.id.back_pressed)
        back_btn.setOnClickListener {
            super@AllCategories.onBackPressed()
        }
    }
}